### ISSUE TITLE
feature suggestion: needle mode

### ISSUE THREAD

**Issue by xuefer** [custom-cards/dual-gauge-card#55]
please add needle mode like https://www.home-assistant.io/dashboards/gauge/

**Comment by mmiller7** [comment]
Yes, I am disappointed this doesn't have a similar option.  Using a needle is far easier to tell at a glance what the max/min and good/bad thresholds are relative to where the current value is.

**Comment by xuefer** [comment]
> Yes, I am disappointed this doesn't have a similar option. Using a needle is far easier to tell at a glance what the max/min and good/bad thresholds are relative to where the current value is.

exactly. "dual" gauge is most needed for climate, to see good/bad threshold and current vs target temperature
let's hope if someone can make "dual" feature into official gauge, or make needle into this dual gauge card
